app/recordtransfer/emails.py

```diff
@@ -49,11 +52,20 @@ def send_submission_creation_success(form_data: dict, submission: Submission) ->
     )
     LOGGER.info("Generated submission change URL: %s", submission_url)
 
-    recipient_emails = _get_admin_recipient_list(subject)
+    admin_recipients = _get_admin_recipient_list(subject)
+    if admin_recipients:
+        used_recipients = admin_recipients
+    elif recipient_emails:
+        used_recipients = recipient_emails
+    else:
+        LOGGER.warning(
+            "No admin recipients found for submission creation success email. Skipping send."
+        )
+        return
```

Comment:
`_get_admin_recipient_list` queries the database - you can avoid calling the database if `recipient_emails` is sent. You can do this by "short-circuiting" if `recipient_emails` is set.

In this example, `_get_admin_recipient_list` is only called if `recipient_emails` is empty or not set.

```suggestion
    used_recipients = recipient_emails or _get_admin_recipient_list(subject)

    if not used_recipients:
        LOGGER.warning(
            "No admin recipients found for submission creation success email. Skipping send."
        )
        return
```